plugins/chain_plugin/chain_plugin.cpp

```diff
@@ -373,6 +373,8 @@ void chain_plugin::set_program_options(options_description& cli, options_descrip
          ("terminate-at-block", bpo::value<uint32_t>()->default_value(0),
           "terminate after reaching this block number (if set to a non-zero number)")
          ("snapshot", bpo::value<bfs::path>(), "File to read Snapshot State from")
+         ("integrity-hash-on-start", bpo::bool_switch(), "Log the state integrity hash on startup")
+         ("integrity-hash-on-stop", bpo::bool_switch(), "Log the state integrity hash on shutdown")
```

Comment:
Seems these could be config.ini options as well.

Reply:
I was waiting to see if anyone would comment on that. My hunch was CLI only but since you mention it I'll move to config